### PR TITLE
Update scope and add template accessors

### PR DESCRIPTION
**Pull request by daniellrgn**
Add scopes for IPS resources
Add placeholder display for new resources

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,6 +64,22 @@ <h1>Patient History</h1>
 
     <div id="patient-info"></div>
     <div id="immunization-history"></div>
+    <div id="AllergyIntoleranceContent"></div>
+    <div id="MedicationStatementContent"></div>
```
Comment (mcjustin) [inline index.html:68]: Not available in Epic R4 UCSDI. Only Medication and MedicationRequest (which you have) are.

https://fhir.epic.com/Documentation?docId=patientfacingfhirapps&section=Appendix

File: index.html [hunk 2]
```diff
@@ -64,6 +64,22 @@ <h1>Patient History</h1>
...
+    <div id="MedicationRequestContent"></div>
+    <div id="MedicationContent"></div>
+    <div id="ConditionContent"></div>
+    <div id="ObservationContent"></div>
+    <div id="OrganizationContent"></div>
+    <div id="DeviceContent"></div>
+    <div id="DeviceUseStatementContent"></div>
```
Comment (mcjustin) [inline index.html:75]: Not available in Epic R4 UCSDI.

File: index.html [hunk 3]
```diff
@@ -64,6 +64,22 @@ <h1>Patient History</h1>
...
+    <div id="DiagnosticReportContent"></div>
+    <div id="ImagingStudyContent"></div>
+    <div id="MediaContent"></div>
```
Comment (mcjustin) [inline index.html:78]: These 2 aren't available in Epic R4 USCDI, in fact I don't see anything image or media -related. Perhaps try "Binary" or "DocumentReference"

File: index.html [hunk 4]
```diff
@@ -64,6 +64,22 @@ <h1>Patient History</h1>
...
+    <div id="PractitionerContent"></div>
+    <div id="PractitionerRoleContent"></div>
+    <div id="ProcedureContent"></div>
+    <div id="SpecimenContent"></div>
```
Comment (mcjustin) [inline index.html:82]: No "Specimen" in Epic R4 USCDI; nothing similar that I see

File: app.js [hunk 5]
```diff
@@ -62,29 +62,281 @@ if (sessionStorage.getItem('SMART_KEY')) { // is there an event like FHIR.oauth2
...
 
-        client.request(`Immunization?patient=${client.getPatientId()}`, { flat: true }).then(immunizations => {
-            immunizationHistory.innerHTML = '<h2>Immunization History:</h2>';
+        requestAndDisplay('Immunization', immunizations => {
+            immunizationHistory.append('<h2>Immunization History:</h2>');
             const list = document.createElement('ul');
 
             for (let i = 0; i < immunizations.length; i++) {
                 const immunization = immunizations[i];
-                if (immunization === undefined || immunization.resourceType != 'Immunization') continue;
```
Comment (mcjustin) [inline app.js:78]: I needed these checks, see https://github.com/uwcirg/sof_uscdi_app/commit/902c6aa1fc8be64badba576c3ff51eafa69fa4b0

File: app.js [hunk 6]
```diff
@@ -62,29 +62,281 @@ if (sessionStorage.getItem('SMART_KEY')) { // is there an event like FHIR.oauth2
...
+        });
+
+        requestAndDisplay('AllergyIntolerance', allergyintolerances => {
+            AllergyIntoleranceContent.append('<h2>Allergies and Intolerances</h2>');
+            const list = document.createElement('ul');
+
+            for (let i = 0; i < allergyintolerances.length; i++) {
+                const allergyintolerance = allergyintolerances[i];
+                const listItem = document.createElement('li');
+                listItem.textContent = "AllergyIntolerance: " + allergyintolerance.fullUrl;
```
Comment (mcjustin) [inline app.js]: Good idea just reading fullUrl for each of these :)

**Review by mcjustin — CHANGES_REQUESTED** [review]

File: app.js (hunk 5 above)
Comment (daniellrgn) [reply app.js:78]: No problem, it's just [pulled out into `requestAndDisplay`](https://github.com/uwcirg/sof_uscdi_app/blob/26476da145d9dc72a51c52afe74f648d340bb8de/app.js#LL87C22-L87C22) so it applies to each of these requests.

**Pull request merged**